sites/brussel/macros.tex

```diff
@@ -1,17 +1,17 @@
 % University Definitions
 \macro{\sitename}{brussel}
-\macro{\university}{Free University of Brussels}
-\macro{\wayf}{VUB -- Vrije Universiteit Brussel}
-\macro{\association}{Brussels University Association (BUA)}
+\macro{\university}{Vrije Universiteit Brussel}
+\macro{\wayf}{VUB}
+\macro{\association}{Universitaire Associatie Brussel}
 % HPC definitions
 \macro{\hpc}{VUB-HPC}
 \macro{\hpcInfra}{VUB-HPC}
 \macro{\hpcTeam}{VUB-HPC}
 \macro{\hpcname}{hydra}
 \macro{\loginnode}{hydra.vub.ac.be}
-\macro{\loginhost}{nic111.hydra.vub.ac.be}
+\macro{\loginhost}{nic50.hydra.vub.ac.be}
 \macro{\loginfingerprint}{41:68:9b:c2:f0:1c:d3:62:14:b6:71:f0:36:8a:7d:ec}
-\macro{\operatingsystem}{Scientific Linux release 6.4 (Carbon)}
+\macro{\operatingsystem}{CentOS Linux release 7.3}
```

Comment:
It's going to 7.4 next week.

Reply:
Yes, next week...

Reply:
Well, I'm not merging your PR this week 😉

Reply:
fine :)